vignettes/datatable-intro.Rmd

```diff
@@ -250,6 +250,50 @@ The function `length()` requires an input argument. We just need to compute the
 
 This type of operation occurs quite frequently, especially while grouping (as we will see in the next section), to the point where `data.table` provides a *special symbol* `.N` for it.
 
+### g) Handle non-existing elements in `i`
```

Comment:
There is already g) for the Basics section ([line 272](https://github.com/Rdatatable/data.table/blob/ff900d1e6a8bcfaa0385bd1304af7b90657d3c4d/vignettes/datatable-intro.Rmd#L272))

Reply:
Make sure the entire document is consistent